src/work/ZeynepAbes.js

```diff
@@ -1,7 +1,87 @@
 import React from 'react';
 
+import Nav from '../Components/Nav';
+import ProjectHeader from '../Components/ProjectHeader';
+import ArtistBio from '../Components/ArtistBio';
+import ProjectColumns from '../Components/ProjectColumns';
+import Video from '../Components/Video';
+
 export default function() {
     return (
-        <h3>The wonderful work of Zeynep Abes</h3>
+        <div className="artist">
+            <Nav />
+            <ProjectHeader
+                artistName="Zeynep Abes"
+                title="Memory Place"
+                materials="Video 1: Photogrammetry of Istiklal Street,<br>
```

Comment:
Unfortunately we can't have html elements in strings here. If you look at the page it renders the `<br>` as plaintext. I think `\n` might work here, but I can't think of how to solve it off the top of my head. One possibility is exposing child elements (similar to my suggestion for the columns component) as part of `ProjectHeader`. These can have style applied and also accept rich html.

Reply:
i can fix this.